light_node/etc/tezedge/tezedge.config

```diff
@@ -30,6 +30,13 @@
 # --bootstrap-db-path <PATH>
 --bootstrap-db-path=bootstrap_db
 
+# Path to timing database directory
+# In case it starts with "./" or "../", it is relative path to the current dir, otherwise to the --tezos-data-dir
+# If directory does not exists, it will be created. If directory already exists, and contains valid database, node
+# will erase it and create a new database
+# --bootstrap-db-path <PATH>
+--timing-db-path=timing_db
```

Comment:
This comment doesn't match the one from `light_node/etc/tezedge/tezedge_drone.config`, and it mentions a different flag.